src/ClassicUO.Assets/MapLoader.cs

```diff
@@ -609,36 +609,36 @@ public void SanitizeMapIndex(ref int map)
     }
 
     [StructLayout(LayoutKind.Sequential, Pack = 1)]
-    public ref struct StaticsBlock
+    public readonly struct StaticsBlock
```

Comment:
Maybe a `readonly ref struct`?

Reply:
The `ref` prevents it from being used in `ReadOnlySpan`, that's why I removed it. You commented on the total PR diff, which is misleading you into believing that I simply replaced `ref` with `readonly`. Look at the individual commit diffs.